models/location_helpers_test.go

```diff
@@ -442,153 +453,3 @@ func TestCreateBackupLocationValidation(t *testing.T) {
 		})
 	}
 }
-
-func TestVerifyBackupLocationValidation(t *testing.T) {
```

Comment:
Why you removed all these tests?

Reply:
Ah, replaced. Ok, nevermind

Reply:
moved below as I see

Reply:
Yeah, because this method was deleted from models